http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -1008,7 +1012,8 @@ object RuntimeServiceInterp {
                 diskConfig,
                 CreateDiskRequest.fromDiskConfigRequest(req, Some(targetZone)),
                 ctx.now,
-                if (willBeUsedBy == FormattedBy.Galaxy) true else false
+                if (willBeUsedBy == FormattedBy.Galaxy) true else false,
```

Comment:
oportunity to clean this up:
```suggestion
                willBeUsedBy == FormattedBy.Galaxy,
```